File: Pattern/pattern3.c
/*
Input N= 5
1
1  2
1  2  3
1  2  3  4
1  2  3  4  5
1  2  3  4
1  2  3
1  2
1

*/
#include <stdio.h>

int main()
{
    int n, row, col;
    printf("Input N= ");
    scanf("%d", &n);

    for (row = 1; row <= n; row++)
    {
        for (col = 1; col <= row; col++)
            printf("%d  ", col);
        printf("\n");
    }
    
        for (row = n-1; row >= 1; row--)
        {
            for (col = 1; col <= row; col++)
                printf("%d  ", col);
            printf("\n");
        }
    
    return 0;
}

File: Pattern/Pattern3.py
n = int(input("Enter the row: "))
for row in range(1, n+1, 1):
    for col in range(1, row+1, 1):
        print(col, end=" ")
    print("\r")

for row in range(n-1, 0, -1):
    for col in range(1, row+1, 1):
        print(col, end=" ")
    print("\r")